plugins/woocommerce/client/blocks/assets/js/blocks/featured-items/inspector-controls.tsx

```diff
@@ -326,6 +364,10 @@ export const withInspectorControls =
 					setAttributes={ setAttributes }
 					setGradient={ setGradient }
 					showDesc={ showDesc }
+					bgColorVisibility={ isBgVisible }
```

Comment:
Let's rename `bgColorVisibility` to `backgroundColorVisibilityStatus` for consistency.